src/astUtils/reflection.ts

```diff
@@ -191,56 +195,49 @@ export function isFunctionParameterExpression(element: Expression | Statement):
     return element?.constructor.name === 'FunctionParameterExpression';
 }
 
-// Value/Type reflection
-
-export function isInvalid(value: any): value is BrsInvalid {
-    return value?.kind === ValueKind.Invalid;
-}
-export function isBoolean(value: any): value is BrsBoolean {
-    return value?.kind === ValueKind.Boolean;
+// BscType reflection
+export function isStringType(value: any): value is StringType {
+    return value?.constructor.name === StringType.name;
 }
-export function isString(value: any): value is BrsString {
```

Comment:
Removed the ValueKind and BrsXYZ reflection functions in favor of common BscType checks instead.